Bixal/move.mil
# Content fixtures behat

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Add behat testing for PPM tool and Entitlements form
- Add missing id, name, label tags for the form fields because Behat needs them
- Address PPM tool back-end bugs

## Testing

To verify the changes proposed in this pull request…

1. pull changes
2. run `circleci build --job behat`
